### PR TITLE
Create deprecated products-and-formats

### PR DESCRIPTION
**Pull request by pgaudet**
What do you think ? This way we can remove any out-of-date info from other pages.

### CODE REVIEW

File: docs/_docs/deprecated-products-and-formats [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Ontology
+<!--OWL RDF/XML-->
+OBO-XML and FASTA files were retired in 2018.
+
+## Annotations
+GO Annotation File Format 1.0 (http://geneontology.org/page/go-annotation-file-gaf-format-10)
+
+## Databases and schemas
+LEAD database
```
Comment (kltm) [inline docs/_docs/deprecated-products-and-formats:11]: Remove the LEAD line

File: docs/_docs/deprecated-products-and-formats [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+## Ontology
+<!--OWL RDF/XML-->
+OBO-XML and FASTA files were retired in 2018.
+
+## Annotations
+GO Annotation File Format 1.0 (http://geneontology.org/page/go-annotation-file-gaf-format-10)
+
+## Databases and schemas
+LEAD database 
+GO MySQL database
```
Comment (kltm) [inline docs/_docs/deprecated-products-and-formats:12]: "MySQL database dumps"

File: docs/_docs/deprecated-products-and-formats [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+Please contact go-help if you have any questions.
```
Comment (kltm) [inline docs/_docs/deprecated-products-and-formats:15]: "Please contact the <a href="http://help.geneontology.org">GO Helpdesk</a> if you have any questions."

File: docs/_docs/deprecated-products-and-formats [hunk 4]
```diff
@@ -0,0 +1,15 @@
+GO is no longer using the following products and formats:
```
Comment (kltm) [inline docs/_docs/deprecated-products-and-formats:1]: "The GOC no longer produces the following files and formats:"

File: docs/_docs/deprecated-products-and-formats [hunk 5]
```diff
@@ -0,0 +1,15 @@
+GO is no longer using the following products and formats: 
+
+## Ontology
+<!--OWL RDF/XML-->
+OBO-XML and FASTA files were retired in 2018.
```
Comment (kltm) [inline docs/_docs/deprecated-products-and-formats:5]: See text in #45

**Comment by pgaudet** [comment]
@kltm I dont understand what I was supposed to do - were these edits suggestions?

**Comment by kltm** [comment]
Suggestions and fixes, part of the requested review.

**Comment by pgaudet** [comment]
OK thanks, I made the changes .

If we remove LEAD database from here, does that means  it is still active ? Do we need to add documentation?

**Comment by kltm** [comment]
LEAD is the MySQL database.

**Comment by pgaudet** [comment]
OK thanks

**Pull request merged**